source/AI.cpp

```diff
@@ -675,7 +675,8 @@ void AI::Step(Command &activeCommands)
 			// Each ship only switches targets twice a second, so that it can
 			// focus on damaging one particular ship.
 			targetTurn = (targetTurn + 1) & 31;
-			if(targetTurn == step || !target || target->IsDestroyed() || (target->IsDisabled() && personality.Disables())
+			if(targetTurn == step || !target || target->IsDestroyed() || (target->IsDisabled() &&
+					(personality.Disables() || target->CanBeCarried()))
```

Comment:
```suggestion
					(personality.Disables() || (target->CanBeCarried() && !personality.IsVindictive())))
```

idk about this one, maybe?

Reply:
I'll leave this one up to the balance folks

Reply:
Vindictive is a rarely used personality, so it's not really a balance concern, more a thematic concern with how vindictive is supposed to behave.
> vindictive: this ship continues to fire at its target even after the target is dead, while the target is beginning to explode.